features/networking/multus.feature

```diff
@@ -7,12 +7,12 @@ Feature: Multus-CNI related scenarios
     # Make sure that the multus is enabled
     Given the master version >= "4.0"
     And the multus is enabled on the cluster
-
+    Given the default interface on the node is stored in the :default_interface clipboard
     # Create the net-attach-def via cluster admin
     Given I have a project
-    When I run the :create admin command with:
-      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/macvlan-bridge.yaml |
-      | n | <%= project.name %> |
+    When I run oc create as admin over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/macvlan-bridge.yaml" replacing paths:
+      | ["metadata"]["namespace"] | <%= project.name %> |    
+      | ["spec"]["config"]["eth0"]| <%= cb.default_interface %> |
```

Comment:
Is the replacement correct here?
It looks a little weird to me. 

The contents under the spec.config should be treated as a single string.

Reply:
yes, see logs:
 [07:24:49] INFO> {"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"macvlan-bridge","namespace":"jivvv"},"spec":{"config":"{ \"cniVersion\": \"0.3.0\", \"type\": \"macvlan\", \"master\": \"ens3\", \"mode\": \"bridge\", \"ipam\": { \"type\": \"host-local\", \"subnet\": \"10.1.1.0/24\", \"rangeStart\": \"10.1.1.100\", \"rangeEnd\": \"10.1.1.200\", \"routes\": [ { \"dst\": \"0.0.0.0/0\" } ], \"gateway\": \"10.1.1.1\" } }"}}

Reply:
Woo, that is amazing!

Reply:
Yep, the logs a lot. ens3 , seems like it grabbed the public cluster default interface. LGTM